packages/eas-cli/src/commandUtils/EasCommand.ts

```diff
@@ -200,7 +200,14 @@ export default abstract class EasCommand extends Command {
           const requestIdLine = graphQLError.extensions?.requestId
             ? `\nRequest ID: ${err.graphQLErrors[0].extensions.requestId}`
             : '';
-          return `${messageLine}${requestIdLine}`;
+
+          const defaultMsg = `${messageLine}${requestIdLine}`;
+
+          if (err?.graphQLErrors?.[0]?.extensions?.errorCode === 'UNAUTHORIZED_ERROR') {
```

Comment:
You should check the `errorCode` of the current error in the mapping loop, so `graphQLError.extensions?.errorCode` instead of checking just the first one - if the first one is `UNAUTHORIZED_ERROR` you'd have your custom line displayed for all the other errors, and if any other than the first one is `UNAUTHORIZED_ERROR` then it won't be displayed at all

Reply:
Oh my God you're absolutely right ! Excuse me I didn't pay close attention, as it made me a bit confused 😵

Reply:
I suggest to try to simplify some parts in code for further maintenance, also, I'd write some guidelines in a README to help contributors understand new updates and workflow.